### PR TITLE
MRC-2077: Queue calibrate tasks as higher priority than model run

### PR DESCRIPTION
**Pull request by r-ash**
Depends on https://github.com/mrc-ide/rrq/pull/27 being merged first

### CODE REVIEW

File: R/main.R [hunk 1]
```diff
@@ -40,7 +40,8 @@ hintr_worker [<queue_id>]"
 main_worker <- function(args = commandArgs(TRUE)) {
   # nocov start
   rrq::rrq_worker(hintr_queue_id(main_worker_args(args)$queue_id, TRUE),
-                  heartbeat_period = 10)
+                  heartbeat_period = 10,
```
Comment (richfitz) [inline R/main.R:43]: We might look at another argument for a worker that *only* does calibrations? That would require changes to the deploy too, and I think worth thinking about in separate tickets/PRs

File: R/queue.R [hunk 2]
```diff
@@ -22,7 +22,8 @@ Queue <- R6::R6Class(
       message(t_("QUEUE_STARTING"))
       queue_id <- hintr_queue_id(queue_id)
       self$queue <- rrq::rrq_controller(queue_id, con)
-      self$queue$worker_config_save("localhost", heartbeat_period = 3)
+      self$queue$worker_config_save("localhost", heartbeat_period = 3,
+                                    queue = c("calibrate", "run"))
```
Comment (richfitz) [inline R/queue.R:26]: Do we have a string constant set anywhere? Replacing these string literals with `queue_calibrate` and `queue_run` (and `queues <- c(queue_calibrate, queue_run)`) might help with some typos

You can also use the default queue for run, which might be simpler?

**Review by richfitz — APPROVED** [review]
Oh wow this is gratifyingly small and clean. Did you try bringing up the whole lot and seeing if it works in practice as this seems too good to be true :)

**Review by richfitz — CHANGES_REQUESTED** [review]
sorry, butterfingers

File: R/main.R (hunk 1 above)
Comment (r-ash) [reply R/main.R:43]: Added a ticket https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-2099

**Comment by r-ash** [comment]
Looks like integration test is flaky in most recent run which is a bit odd.

**Comment by r-ash** [comment]
Have confirmed that I can see this working locally

**Review by richfitz — APPROVED** [review]

**Pull request merged**